homeassistant/components/tasmota/sensor.py

```diff
@@ -162,7 +176,7 @@
     hc.SENSOR_TODAY: {DEVICE_CLASS: SensorDeviceClass.ENERGY},
     hc.SENSOR_TOTAL: {
         DEVICE_CLASS: SensorDeviceClass.ENERGY,
-        STATE_CLASS: SensorStateClass.TOTAL_INCREASING,
+        #        STATE_CLASS: SensorStateClass.TOTAL_INCREASING,
```

Comment:
We should set a state class, or no statistics will be generated.
It should be state class TOTAL, with or without setting last_reset: https://developers.home-assistant.io/docs/core/entity/sensor/#how-to-choose-state_class-and-last_reset